.one-pipeline.yaml

```diff
@@ -62,6 +62,14 @@ test:
   script: |
     #!/usr/bin/env bash
 
+    PERIODIC_TRIGGER_RUN=$(get_env periodic-trigger-run)
```

Comment:
There could be multiple periodic triggers in the future for different purposes. Suggest `periodic-rescan`